subsys/caf/modules/Kconfig.buttons

```diff
@@ -23,6 +23,13 @@ config CAF_BUTTONS_PM_EVENTS
 	depends on CAF_PM_EVENTS
 	default y
 
+config CAF_BUTTONS_PM_KEEP_ALIVE
+	bool "Generate keep alive power event on button press"
+	depends on CAF_KEEP_ALIVE_EVENTS
+	default y
```

Comment:
I am not sure this should be the case in respect of nrf_desktop.

Reply:
I think it could delay sending a HID report (additional operation is added between generating hardware input and sending an HID report). Maybe we should simply disable it for nRF Desktop ?

Reply:
This is the question. If we should keep it enabled by default and disable it in desktop to keep it consistent with others CAF modules or should we make a waiver here and leave it disabled by default and enable when needed?

Reply:
In my opinion we should enable it by default and disable explicitly in nRF Desktop.

CAF configuration should be consistent and I think that handling power management should be enabled by default, because in most cases we want to use it in all of the modules of application that enables power management (we already had some issues related to overlooking Kconfigs when handling power management events was disabled by default).
If user wants to enable some optimizations (like in nRF Desktop), then Kconfig options can be used.

Reply:
ok